panel/template/fast/base.py

```diff
@@ -12,7 +12,7 @@
 
 class FastBaseTemplate(BasicTemplate):
 
-    accent_base_color = param.String(doc="""
+    accent_base_color = param.String(default="#0072B5", doc="""
```

Comment:
I wanted a way for the user to get the color for their app to make it easier to use the color in plots etc.